crates/invoker-api/src/status_handle.rs

```diff
@@ -25,6 +25,8 @@ pub struct InvocationStatusReportInner {
     pub start_count: usize,
     pub last_start_at: SystemTime,
     pub last_retry_attempt_failure: Option<InvocationErrorReport>,
+    pub next_retry_at: Option<SystemTime>,
+    pub last_attempt_endpoint_id: Option<EndpointId>,
```

Comment:
I'm not sure why you would add the `last_attempt_endpoint_id` both in the `sys_invocation_state` and in `sys_status`. For this field, the source of truth is `sys_status` anyway.

Reply:
Ah ok i see. You can track the case where you tried to use the endpooint_id but has not been committed yet. makes sense. Perhaps let's add a comment about that somewhere in the schema of the sys_invocation_state table, to avoid me making the same comment in 3 months :)